Sidebar responsiveness

FelixLoether opened the pull request:
Closes #35.

Built on top of #34, so that should be merged first.

lraivio commented on app/screens/sidebar/ownerFilter/ownerFilter.less:
```diff
@@ -23,3 +23,11 @@
   }
 }
+
+@media (max-width: 768px) {
```
- [ ] The 768px limit for mobile view could be stored into a variable.

FelixLoether commented:
Moved 768px to a variable. I also removed the commit that specified a height for the select multiple boxes on mobile. I realized that if a non-mobile has a narrow window, the select boxes would become kind of unusable with those styles due to having such a small height. Now they don't look quite as good on mobile devices, but they are usable on all devices.
